index.ts

```diff
@@ -0,0 +1,394 @@
+/* global Promise */
+import axeCore = require('axe-core');
+import rIC = require('requestidlecallback');
+import after = require('./after');
+
+// @reference https://www.npmjs.com/package/@types/requestidlecallback
+// (it's declared as type=dom so had to copy/paste here)
```

Comment:
What does "it's declared as type=dom" mean? Can't we just add `"dom"` to `lib: []`?